README.md

```diff
@@ -1,24 +1,43 @@
-# README
+# Project: Budjet App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Built With
```

Comment:
* Kindly add a live version link of your project as it's [required](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=The%20project%20should%20be%20deployed%20and%20accessible%20online.) that The project should be deployed and accessible online.